File: Submissions/T13/H2.py
n, m = map(int, input("Baris & Kolom: ").split())
matrix = []

for i in range(n):
  baris = list(map(int, input (f"Baris {i + 1}: ").split()))
  matrix.append(baris)

print(f"Matriks ({n}x{m}): ")
for baris in matrix:
  print(" ".join(map(str, baris)))

print(f"Transpose ({m}x{n}): ")
for j in range(m):
  transpose = [matrix[i][j] for i in range(n)]
  print(" ".join(map(str, transpose)))

File: Submissions/T06/H2.cpp
#include <stdio.h>

int main() {
    int N, M;

    printf("Baris & Kolom : ");
    scanf("%d %d", &N, &M);

    int matriks[N][M];

    for (int i = 0; i < N; i++) {
        printf("Baris %d : ", i + 1);
        for (int j = 0; j < M; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }

    printf("\n");

    printf("Matriks (%dx%d) :\n", N, M);
    for (int i = 0; i < N; i++) {
        for (int j = 0; j < M; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }

    
    printf("Transpose (%dx%d) :\n", M, N);
    for (int j = 0; j < M; j++) {
        for (int i = 0; i < N; i++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }

    return 0;
}
